modules/notification/webhook/webhook.go

```diff
@@ -310,8 +310,11 @@ func (m *webhookNotifier) NotifyNewPullRequest(pull *models.PullRequest, mention
 }
 
 func (m *webhookNotifier) NotifyIssueChangeContent(doer *models.User, issue *models.Issue, oldContent string) {
-	mode, _ := models.AccessLevel(issue.Poster, issue.Repo)
 	var err error
+	if err = issue.LoadRepo(); err != nil {
+		log.Error("NotifyIssueChangeContent: coulnd't load repo", err)
```

Comment:
Then just return, otherwise it will panic.

Reply:
Typo: coulnd't